dev/README_RELEASE_AIRFLOW.md

```diff
@@ -314,12 +314,12 @@ The Release Candidate artifacts we vote upon should be the exact ones we vote ag
 
     ```shell script
     # First clone the repo
-    svn checkout https://dist.apache.org/repos/dist/dev/airflow airflow-dev
-    cd airflow-dev
-    # Or move into it if you already have it cloned
+
+    [ -d asf-dist ] || svn checkout --depth=immediates https://dist.apache.org/repos/dist asf-dist
```

Comment:
I like the idea of unifying it iunder arlfow repo. We need to add asf-dist to .gitignore though.

Reply:
Oh, I've always just run this from the dir above the `airflow` git checkout.

Reply:
Not what the current instructions tell you to do :) 


```
cd <YOUR_AIRFLOW_REPO_ROOT>
export AIRFLOW_REPO_ROOT=$(pwd)
[ -d asf-dist ] || svn checkout --depth=immediates https://dist.apache.org/repos/dist asf-dist
svn update --set-depth=infinity asf-dist/{release,dev}/airflow
AIRFLOW_DEV_SVN="${PWD}/asf-dist/dev/airflow"
cd asf-dist/release/airflow
``

Reply:
Oh yeah. Any objections to me changing that too?

Reply:
no problem